.github/workflows/test.yml

```diff
@@ -416,3 +478,67 @@ jobs:
           make test
           ./test
         working-directory: ./stdlib
+
+  coverage:
+    runs-on: ubuntu-latest
+    name: Coverage report
+    container: ghcr.io/hyperledger/solang-llvm:ci-5
+    needs: [linux-x86-64, solana, anchor, polkadot, polkadot-subxt, vscode]
+    env:
+      RUSTFLAGS: -C instrument-coverage -C llvm-args=--instrprof-atomic-counter-update-all --cfg=coverage --cfg=trybuild_no_target
+      LLVM_PROFILE_FILE: ${{ github.workspace }}/target/solang-%p-%10m.profraw
+      CARGO_INCREMENTAL: 0
+      CARGO_LLVM_COV: 1
+      CARGO_LLVM_COV_TARGET_DIR: ${{ github.workspace }}/target
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: llvm-tools
```

Comment:
You're not specify the toolchain version - this may cause problems. i.e. build with version X, but now the prof files won't work with X + 1

Reply:
Good catch.